tests/unit/lms/tasks/mailchimp_test.py

```diff
@@ -5,7 +5,28 @@
 import pytest
 
 from lms.services.mailchimp import EmailRecipient, EmailSender
-from lms.tasks.mailchimp import send_template
+from lms.tasks.mailchimp import send, send_template
+
+
+@pytest.mark.usefixtures("mailchimp_service")
+class TestSend:
+    def test_it(self, mailchimp_service):
+        sender = EmailSender("subaccount", "sender_email", "sender_name")
+        recipient = EmailRecipient("recipient_email", "recipient_name")
+
+        send(
+            sender=asdict(sender),
+            recipient=asdict(recipient),
+            template=sentinel.template,
+            template_vars=sentinel.template_vars,
+        )
+
+        mailchimp_service.send.assert_called_once_with(
+            sender=sender,
+            recipient=recipient,
+            template=sentinel.template,
+            template_vars=sentinel.template_vars,
+        )
```

Comment:
This is just the `TestSendTemplate` class below copy-pasted and adjusted to test `send()` instead of `send_template()`